pytorch_lightning/utilities/imports.py

```diff
@@ -72,26 +100,26 @@ def _compare_version(package: str, op: Callable, version: str, use_base_version:
 # _TORCH_GREATER_EQUAL_DEV_1_11 = _compare_version("torch", operator.ge, "1.11.0", use_base_version=True)
 
 _APEX_AVAILABLE = _module_available("apex.amp")
-_DEEPSPEED_AVAILABLE = _module_available("deepspeed")
+_DEEPSPEED_AVAILABLE = _package_available("deepspeed")
 _FAIRSCALE_AVAILABLE = not _IS_WINDOWS and _module_available("fairscale.nn")
 _FAIRSCALE_OSS_FP16_BROADCAST_AVAILABLE = _FAIRSCALE_AVAILABLE and _compare_version("fairscale", operator.ge, "0.3.3")
 _FAIRSCALE_FULLY_SHARDED_AVAILABLE = _FAIRSCALE_AVAILABLE and _compare_version("fairscale", operator.ge, "0.3.4")
 _GROUP_AVAILABLE = not _IS_WINDOWS and _module_available("torch.distributed.group")
-_HOROVOD_AVAILABLE = _module_available("horovod.torch")
-_HYDRA_AVAILABLE = _module_available("hydra")
+_HOROVOD_AVAILABLE = _package_available("horovod.torch")
```

Comment:
Should this be module?

```suggestion
_HOROVOD_AVAILABLE = _module_available("horovod.torch")
```